package.json

```diff
@@ -9,6 +9,10 @@
   "scripts": {
     "prettier:check": "prettier --check .",
     "prettier:fix": "prettier --write .",
+    "lint:check": "pnpm -r lint:check",
+    "lint:fix": "pnpm -r lint:fix",
+    "code:checks": "pnpm lint:check ; pnpm lint:check",
```

Comment:
```suggestion
    "code:checks": "pnpm lint:check && pnpm lint:check",
```

Reply:
I used `;` so that you can see all the failures instead of stopping to the lint ones, but if you prefer I'm happy with `&&` 🙂

Reply:
Oh nice... TIL